code/modules/food_and_drinks/food.dm

```diff
@@ -40,4 +40,6 @@
 				if(foodtype & H.dna.species.toxic_food)
 					to_chat(H, "<span class='warning'>You don't feel so good...</span>")
 					H.adjust_disgust(25 + 30 * fraction)
+			if(foodtype & BREAKFAST && world.time - SSticker.round_start_time < 9000)
```

Comment:
This really ought to be a define, and it should be 15 MINUTES

Reply:
ok thanks, to be honest I was taking code from else where in the code because I dont know how time works in dm code too well